.travis.yml

```diff
@@ -33,6 +35,20 @@ install:
     - git remote add riot https://github.com/RIOT-OS/RIOT.git
     - git fetch riot master
     - git log -1 --pretty=format:%H riot/master
+    - echo "$PATH"
+    - ln -s /usr/bin/ccache /usr/local/bin/gcc
+    - ln -s /usr/bin/ccache /usr/local/bin/g++
+    - ln -s /usr/bin/ccache /usr/local/bin/cc
+    - ln -s /usr/bin/ccache /usr/local/bin/c++
+    - ln -s /usr/bin/ccache /usr/local/bin/arm-none-eabi-gcc
+    - ln -s /usr/bin/ccache /usr/local/bin/arm-none-eabi-g++
+    - ln -s /usr/bin/ccache /usr/local/bin/arm-none-eabi-c++
+    - ln -s /usr/bin/ccache /usr/local/bin/avr-gcc
+    - ln -s /usr/bin/ccache /usr/local/bin/avr-g++
+    - ln -s /usr/bin/ccache /usr/local/bin/avr-c++
+    - ln -s /usr/bin/ccache /usr/local/bin/msp430-gcc
+    - ln -s /usr/bin/ccache /usr/local/bin/msp430-g++
+    - ln -s /usr/bin/ccache /usr/local/bin/msp430-c++
```

Comment:
The ccache documentation is a little bit misleading (for debian/ubuntu users). AFAIK you don't have to create those symlinks by hand. On a debian/ubuntu system there is a directory (```/usr/lib/ccache```) already populated with symlinks for all installed compilers on your system. You just have prepend that directory to your PATH.